examples/azure-functions-http-trigger-with-queue/azure-functions-http-trigger-with-queue.md

```diff
@@ -0,0 +1,43 @@
+# Azure Functions- HTTP trigger with queue
```

Comment:
```suggestion
# Azure Functions - HTTP trigger with queue
```

Reply:
Also, some files have this hyphen while others do not. I prefer the version with a hyphen. Check all the files and be consistent.

Reply:
Fixed.